### PR TITLE
Addition of Missing "Digital Pins" link.

### PR DESCRIPTION
**Pull request by animeshsrivastava24**
Addition of Missing "Digital Pins" link.

### CODE REVIEW

File: Language/Functions/Digital IO/digitalWrite.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ Write a `HIGH` or a `LOW` value to a digital pin.
 If the pin has been configured as an `OUTPUT` with `pinMode()`, its voltage will be set to the corresponding value: 5V (or 3.3V on 3.3V boards) for `HIGH`, 0V (ground) for `LOW`.
 [%hardbreaks]
 
-If the pin is configured as an `INPUT`, `digitalWrite()` will enable (`HIGH`) or disable (`LOW`) the internal pullup on the input pin. It is recommended to set the `pinMode()` to `INPUT_PULLUP` to enable the internal pull-up resistor. See the digital pins tutorial for more information.
+If the pin is configured as an `INPUT`, `digitalWrite()` will enable (`HIGH`) or disable (`LOW`) the internal pullup on the input pin. It is recommended to set the `pinMode()` to `INPUT_PULLUP` to enable the internal pull-up resistor. See the http://arduino.cc/en/Tutorial/DigitalPins[Digital Pins] tutorial for more information.
```
Comment (per1234) [inline Language/Functions/Digital IO/digitalWrite.adoc]:
```suggestion
If the pin is configured as an `INPUT`, `digitalWrite()` will enable (`HIGH`) or disable (`LOW`) the internal pullup on the input pin. It is recommended to set the `pinMode()` to `INPUT_PULLUP` to enable the internal pull-up resistor. See the http://arduino.cc/en/Tutorial/DigitalPins[Digital Pins^] tutorial for more information.
```
From the reference sample:
https://github.com/arduino/reference-en/blob/afde77c8537eb03d60ef066011bdbd425722c90a/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc#L127-L128
As you can see from the example, "external links" refers to any page outside the Arduino Language Reference.

Comment (animeshsrivastava24) [reply Language/Functions/Digital IO/digitalWrite.adoc]: Thanks a lot for this information, I'll keep this in mind while doing any modifications related to this in future.

**Comment by animeshsrivastava24** [comment]
@per1234  Resolved the conversation, please have a view.

File: Language/Functions/Digital IO/digitalWrite.adoc (hunk 1 above)
Comment (per1234) [reply Language/Functions/Digital IO/digitalWrite.adoc]: @animeshsrivastava24 the issue is not resolved. I requested that you add the carat to the end of the link text for consistency with the standard established by the reference sample, but you have not done so.

Comment (animeshsrivastava24) [reply Language/Functions/Digital IO/digitalWrite.adoc]: Please review now, it might have been missed accidentally.

**Comment by per1234** [comment]
Thanks!

**Pull request merged**